bundle/Templating/Twig/Extension/NamedObjectRuntime.php

```diff
@@ -21,33 +23,66 @@ class NamedObjectRuntime
      */
     private $namedObjectProvider;
 
-    public function __construct(Provider $specialObjectProvider)
-    {
+    /**
+     * @var \Psr\Log\LoggerInterface
+     */
+    private $logger;
+
+    /**
+     * @var bool
+     */
+    private $isDebug;
+
+    public function __construct(
+        Provider $specialObjectProvider,
+        LoggerInterface $logger,
+        bool $isDebug
+    ) {
         $this->namedObjectProvider = $specialObjectProvider;
+        $this->logger = $logger;
+        $this->isDebug = $isDebug;
     }
 
     public function getNamedContent(string $name): ?Content
     {
-        if ($this->namedObjectProvider->hasContent($name)) {
-            return $this->namedObjectProvider->getContent($name);
+        try {
+            if ($this->namedObjectProvider->hasContent($name)) {
+                return $this->namedObjectProvider->getContent($name);
+            }
+        } catch (UnauthorizedException $e) {
+            if ($this->isDebug) {
+                $this->logger->critical($e->getMessage());
```

Comment:
We can use `debug` here, since we're now logging it only in debug mode.

Reply:
Updated: https://github.com/netgen/ezplatform-site-api/pull/164/commits/0a7d31ee8fd53538aecd9263d0a2ce90f7da14e8